cdap-ui/app/cdap/components/FastAction/SetPreferenceAction/SetPreferenceModal.js

```diff
@@ -403,20 +402,20 @@ export default class SetPreferenceModal extends Component {
                       className="btn btn-primary float-xs-left"
                       disabled="disabled"
                     >
-                      <span className="fa fa-spinner fa-spin"></span>
+                      <span className="fa fa-spinner fa-spin" />
```

Comment:
I'm not sure if you can actually do this... I don't think you can use self-closing tags on non-void HTML elements.